### ISSUE TITLE
Incompatibility with dash 1.21.0 when serving files locally

### ISSUE THREAD

**Issue by radusuciu** [GibbsConsulting/django-plotly-dash#349]
When serving files locally after upgrading to dash 1.21.0, certain script urls break. This is what is rendered in the template footer:

```
<script src="/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash/deps/polyfill@7.v1_21_0m1627189189.12.1.min.js"></script>
<script src="/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash/deps/react@16.v1_21_0m1627189189.14.0.min.js"></script>
<script src="/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash/deps/react-dom@16.v1_21_0m1627189189.14.0.min.js"></script>
<script src="/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash/deps/prop-types@15.v1_21_0m1627189189.7.2.min.js"></script>
<script src="/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash/dash-renderer/build/dash_renderer.v1_21_0m1627189189.min.js"></script>
```

Note the `/dash/deps/` bit, which is not handled by any route. With dash 1.20.0, the folder structure is different, eg. for `react-dom`, it is:

```
<script src="/synapse/django_plotly_dash/app/LigandabilityViewer/_dash-component-suites/dash_renderer/react-dom@16.v1_9_1m1617985068.14.0.min.js"></script>
```

**Comment by GibbsConsulting** [comment]
Sounds like a code change in dash 1.21.0 has altered some paths. As `django-plotly-dash` rewrites some of these paths we will need to update our codebase.

Short term workaround is to restrict the dash version used, either manually or in a project's requirements file.

**Comment by radusuciu** [comment]
Thanks for investigating!

**Comment by GibbsConsulting** [comment]
@radusuciu v1.6.5 has been released, which provides a workaround by restricting the permissible dash version.
A more complete fix will require #351 or equivalent - this will allow the version restriction to be lifted

**Comment by radusuciu** [comment]
Thank you again!

**Comment by GibbsConsulting** [comment]
@weber-s just released v1.6.6 which should resolve this. If you could check again and reopen if the issue persists that would be great.
